pkg/velaql/providers/query/handler_test.go

```diff
@@ -962,19 +974,18 @@ options: {
 			"http://text.example.com",
 			"10.10.10.10:81",
 			"text.example.com:81",
-			// helmRelease
-			fmt.Sprintf("http://%s:30002", gatewayIP),
```

Comment:
Why remove the rules from the helm release?

Reply:
it not work in tests